languages/ruby/exercises/concept/exceptions/simple_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
```

Comment:
The `-` operator is not listed as a supported operation. I don't mind having this in the stub file, although you could also consider removing it. The benefit of having it in is that there will be less variance in the submitted solutions, which helps with the representation matching.